ui/ui_vfinal/intents/javascript_functions/submit_query.html

```diff
@@ -208,6 +174,109 @@
       document.querySelector('#debiasing-suggestions').classList.add('hide');
     }
   }
+  
+  
+  // helper functions to show & display next & previous button
+
+  function hideNextButton() {
+    document.querySelector('#next').classList.add('hide');
+  }
+  
+  function displayNextButton() {
+    document.querySelector('#next').classList.remove('hide');
+  }
+  
+  function hidePreviousButton() {
+    document.querySelector('#previous').classList.add('hide');
+  }
+  
+  function displayPreviousButton() {
+    document.querySelector('#previous').classList.remove('hide');
+  }
+  
+  function showNext() {
+      index++;
+      if(index >= suggestions.length - 1) {
+          // hide next button
+          hideNextButton();
+      }
+      else {
+          displayNextButton();
+      }
+    
+      if(index >= suggestions.length) 
```

Comment:
Let's follow the consistent style throughout the code : 
if (cond) {
  // Some code
} else {
  // some other code
}

Reply:
Done.

Reply:
Let's do it for this if also. 
if (index >= suggestions.length) {
  return;
}